integration_test/gce/gce_testing.go

```diff
@@ -1151,12 +1114,21 @@ func attemptCreateInstance(ctx context.Context, logger *log.Logger, options VMOp
 	}
 
 	imgProject := options.ImageProject
+
+	delim := ""
+	if strings.Contains(options.Platform, ":"){
+		delim = ":"
+	} else if strings.Contains(options.Platform, "="){
+		delim = "="
+	} else {
+		return nil, fmt.Errorf("could not parse options.Platform from struct: %+v", options)
+	}
+ 
+	s := strings.Split(options.Platform, delim)
```

Comment:
There are comments, e.g. the one at the top of this file, about what the PLATFORMS env variable should look like and those are all wrong after this PR. They should be updated.

Reply:
c+p from another comment: "Spoken offline, and agree that the long term solution is to replace instances of platform with imageSpec and updated logic. A followup PR will do a sweeping refactor along these lines."